api/src/main/java/marquez/cli/SeedCommand.java

```diff
@@ -0,0 +1,1419 @@
+import lombok.NonNull;
+import lombok.ToString;
+import lombok.extern.slf4j.Slf4j;
+import marquez.MarquezConfig;
+import marquez.client.MarquezClient;
+import marquez.client.Utils;
+import marquez.client.models.Dataset;
+import marquez.client.models.DatasetId;
+import marquez.client.models.DatasetMeta;
+import marquez.client.models.DbTableMeta;
+import marquez.client.models.Field;
+import marquez.client.models.Job;
+import marquez.client.models.JobMeta;
+import marquez.client.models.Namespace;
+import marquez.client.models.NamespaceMeta;
+import marquez.client.models.Run;
+import marquez.client.models.RunMeta;
+import marquez.client.models.Source;
+import marquez.client.models.SourceMeta;
+
+@Slf4j
+public final class SeedCommand extends ConfiguredCommand<MarquezConfig> {
+  static final String DEFAULT_DATAKIN_HOST = "localhost";
+  static final int DEFAULT_DATAKIN_PORT = 8080;
+
+  public static final String NAMESPACE_NAME = "food_delivery";
+  static final String SOURCE_NAME = "analytics_db";
+
+  static final int LINEAGE_GRAPH_24_HOUR_WINDOW = 24;
+  static final int RUN_TIME_IN_SEC_MIN = 120;
+  static final int RUN_TIME_IN_SEC_MAX = 240;
+
+  public SeedCommand() {
+    super("seed", "seeds the HTTP API with metadata");
+  }
+
+  @Override
+  public void configure(@NonNull final net.sourceforge.argparse4j.inf.Subparser subparser) {
+    super.configure(subparser);
+    subparser
+        .addArgument("--host")
+        .dest("host")
+        .type(String.class)
+        .required(false)
+        .setDefault(DEFAULT_DATAKIN_HOST)
+        .help("the HTTP API server host");
+    subparser
+        .addArgument("--port")
+        .dest("port")
+        .type(Integer.class)
+        .required(false)
+        .setDefault(DEFAULT_DATAKIN_PORT)
+        .help("the HTTP API server port");
+  }
+
+  @Override
+  protected void run(
+      @NonNull final Bootstrap<MarquezConfig> bootstrap,
+      @NonNull final net.sourceforge.argparse4j.inf.Namespace namespace,
+      @NonNull final MarquezConfig config) {
+    final String host = namespace.getString("host");
+    final int port = namespace.getInt("port");
+
+    final URL baseUrl = Utils.toUrl(String.format("http://%s:%d", host, port));
+    final MarquezClient client = MarquezClient.builder().baseUrl(baseUrl).build();
+    seed(client, LINEAGE_GRAPH_24_HOUR_WINDOW);
+  }
+
+  public void seed(@NonNull MarquezClient client, int additionalIterations) {
+    // (1) Create namespace
+    final NamespaceMeta namespaceMeta =
+        NamespaceMeta.builder()
+            .ownerName("owner@food.com")
+            .description("Food delivery example!")
+            .build();
+    final Namespace newNamespace = client.createNamespace(NAMESPACE_NAME, namespaceMeta);
+    log.info("Created namespace: {}", newNamespace);
+
+    // (2) Create source
+    final SourceMeta sourceMeta =
+        SourceMeta.builder()
+            .type("MYSQL")
+            .connectionUrl("jdbc:mysql://localhost:3306/deliveries")
```

Comment:
postgres?

Reply:
For the example, the database type isn't as important. But, I'll update because postgres is awesome!